index.md

```diff
@@ -1,4 +1,10 @@
 ---
+title: NAIMS-AIMS
 ---
+## What is NAIMS-AIMS?
+<p align="center" style="font-size:150%">The Artificial Intelligence and Machine-Learning (AIMS) working group of <br/> the North American Imaging in MS (NAIMS) Cooperative</p>
 
-Welcome to the NAIMS-AIMS Website!
+We are a group of academic researchers in the field of medical imaging analysis. The mission of this working group is to help the overall [NAIMS](https://www.naimscooperative.org/) group to leverage AI and ML methods to make better use of neuroimaging data, in order to help achieve NAIMS overall goals of using imaging to improve the lives of people with MS. Learn more about us in our [About Us](/about-us) page.
+
+## News
+* We will be hosting our first (virtual) workshop on Federated Learning on January 20th, 2023. Please join us for a day of informative talks from and discussions with experts in the field. More information can be found on the workshop's [page](/workshops/workshop2023). 
```

Comment:
I assumed you've tested the relative link on your local build, but just in case, I would have written the link as `workshops/workshop2023` instead of `/workshops/workshop2023`